bundles/org.openhab.core.io.transport.serial.rxtx/src/main/java/org/openhab/core/io/transport/serial/internal/RxTxPortProvider.java

```diff
@@ -45,11 +49,19 @@ public class RxTxPortProvider implements SerialPortProvider {
 
     @Override
     public @Nullable SerialPortIdentifier getPortIdentifier(URI port) {
+        String portPathAsString = port.getPath();
         try {
-            CommPortIdentifier ident = SerialPortUtil.getPortIdentifier(port.getPath());
+            // Resolving symbolic link is needed because of a bug with nrjavaserial
+            // Until a new release with pull request #230 is not included in openHAB,
+            // we resolve symbolic link here here
+            Path portPath = Path.of(portPathAsString);
+            if (Files.isSymbolicLink(portPath)) {
+                portPathAsString = portPath.toRealPath().toString();
+            }
+            CommPortIdentifier ident = SerialPortUtil.getPortIdentifier(portPathAsString);
             return new SerialPortIdentifierImpl(ident);
-        } catch (NoSuchPortException e) {
-            logger.debug("No SerialPortIdentifier found for: {}", port.getPath());
+        } catch (NoSuchPortException | IOException e) {
+            logger.debug("No SerialPortIdentifier found for: {}", portPathAsString);
```

Comment:
The exception causing this may contain useful info when debug logging is enabled:

```suggestion
            logger.debug("No SerialPortIdentifier found for: {}", portPathAsString, e);
```

Reply:
Done. Sorry for the delay.
I also did a rebase /squash and so force push.